usaspending_api/etl/management/commands/es_rapidloader.py

```diff
@@ -208,50 +166,103 @@ def controller(self):
         while True:
             sleep(10)
             if process_guarddog(process_list):
-                raise SystemExit(1)
+                raise SystemExit("Fatal error: review logs to determine why process died.")
             elif all([not x.is_alive() for x in process_list]):
                 printf({"msg": "All ETL processes completed execution with no error codes"})
                 break
 
-        if self.config["reload_all"]:
+    def complete_process(self) -> None:
+        if self.config["create_new_index"]:
             printf({"msg": "Closing old indices and adding aliases"})
-            swap_aliases(ES, self.config["index_name"])
+            set_final_index_config(self.elasticsearch_client, self.config["index_name"])
+            swap_aliases(self.elasticsearch_client, self.config["index_name"])
 
         if self.config["snapshot"]:
             printf({"msg": "Taking snapshot"})
-            take_snapshot(ES, self.config["index_name"], settings.ES_REPOSITORY)
+            take_snapshot(self.elasticsearch_client, self.config["index_name"], settings.ES_REPOSITORY)
+
+        if self.config["is_incremental_load"]:
+            printf({"msg": "Updating last load record in DB with {}".format(self.config["processing_start_datetime"])})
```

Comment:
`table.field` is abstracted away from this script. Would either need to import the model or hard-code the tablename and field in this log statement. Neither is better than what is currently there.

Other ETL scripts don't contain this log message. Either I scrub this one to avoid confusion, keep it as-is, or change to it like suggestion:
```Storing datetime {} for next incremental load```